requirements.txt

```diff
@@ -0,0 +1,2 @@
+numpy==1.10.4
+root-numpy==4.7.2
```

Comment:
Last thing probably, but prefer a ```>=``` version check rather than a ```==``` as I doubt a fixed version is going to be mandatory